### PR TITLE
Feat share button

### PR DESCRIPTION
**Pull request by makram93**
Add social share button

### CODE REVIEW

File: deployment/playground/playground.py [hunk 1]
```diff
@@ -90,6 +91,7 @@ def deploy_streamlit():
             # for now deactivated sample images for text
             output_modality_dir = 'text'
             data_dir = root_data_dir + output_modality_dir + '/'
+            print(data_dir + DATA + f'.txt10-{docarray_version}.bin')
```
Comment (joschkabraun) [inline deployment/playground/playground.py:94]: don't need the print

File: deployment/playground/playground.py [hunk 2]
```diff
@@ -58,15 +59,15 @@ def deploy_streamlit():
...
-    HOST = query_parameters.get('host')[0]
-    PORT = (
-        query_parameters.get('port')[0] if 'port' in query_parameters.keys() else None
-    )
-    OUTPUT_MODALITY = query_parameters.get('output_modality')[0]
-    INPUT_MODALITY = query_parameters.get('input_modality')[0]
-    DATA = (
-        query_parameters.get('data')[0] if 'data' in query_parameters.keys() else None
-    )
+    HOST = "grpcs://nowapi-1cf62a2f29.wolf.jina.ai"  # query_parameters.get('host')[0]
```
Comment (joschkabraun) [inline deployment/playground/playground.py:62]: You forgot to comment out the values for testing

File: now/deployment/deployment.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 
 def deploy_wolf(path: str, name: str, env_file: str = None):
-    return CloudFlow(path=path, name=name, env_file=env_file).__enter__()
+    return
```
Comment (joschkabraun) [inline now/deployment/deployment.py:9]: What is the purpose of this? If that is intended, we should remove this function.

**Review by joschkabraun — CHANGES_REQUESTED** [review]
Nice, adding of sharing button!

File: deployment/playground/playground.py (hunk 2 above)
Comment (makram93) [reply deployment/playground/playground.py]: Was testing it locally, forgot to revert this. Thanks for pointing it out

**Pull request merged**